distributed/worker.py

```diff
@@ -97,17 +99,42 @@
 
 no_value = "--no-value-sentinel--"
 
+
+class WorkerTaskStateName(Enum):
+    released = "released"
+    memory = "memory"
+    waiting = "waiting"
+    ready = "ready"
+    constrained = "constrained"
+    executing = "executing"
+    long_running = "long-running"
```

Comment:
This is going to cause headaches with name != value. Can we use _ in both cases?

Reply:
(may need to change on the scheduler side too)

Reply:
I kept it this way for compat purposes. The value is backwards compatible